README.md

```diff
@@ -203,15 +189,15 @@ Finally, the launch command for the simulation application is as follows:
 
 ```bash
 source simulation_ws/install/setup.sh
-roslaunch object_tracker_simulation local_training.launch
+ros2 launch object_tracker_simulation local_training.launch.py
```

Comment:
Why doesn't this need `source /usr/share/gazebo/setup.sh`?

Reply:
I haven't tested this on RoboMaker yet, but the discussion made it sound like `/usr/share/gazebo/setup.sh` doesn't need manual sourcing from the user when in a RoboMaker simulation environment.

Reply:
Okay, makes sense